### PR TITLE
rg-11: Preview content page layout

### PR DESCRIPTION
**Pull request by honkster-nazarii**
Made layout according to design

Previews:
Знімок екрана 2024-02-13 в 00 17 17
Знімок екрана 2024-02-13 в 00 17 40

### CODE REVIEW

**Review by apofis96 — APPROVED** [review]

File: frontend/src/assets/img/resumeIcon.svg
Comment (Vitaliy-Kalenichenko) [inline frontend/src/assets/img/resumeIcon.svg:1]: Set width, height = 100%

File: frontend/src/bundles/home/components/home-top-section/components/create-resume-button/create-resume-button.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import resumeIcon from '~/assets/img/resumeIcon.svg';
+
+import styles from './styles.module.scss';
+
+const CreateResumeButton: React.FC = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.create_resume_button}>
+                <div className={styles.create_resume_button__icon}>
+                    <img
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/home/components/home-top-section/components/create-resume-button/create-resume-button.tsx:11]: Don't use img for svg, you can use svg as a component

**Comment by Vitaliy-Kalenichenko** [comment]
Resolve conflicts

File: frontend/src/bundles/home/components/home-top-section/components/create-resume-button/create-resume-button.tsx (hunk 1 above)
Comment (honkster-nazarii) [reply frontend/src/bundles/home/components/home-top-section/components/create-resume-button/create-resume-button.tsx]: doesn't it make html insanely ugly

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Pull request merged**